mlflow/paddle/__init__.py

```diff
@@ -430,3 +444,190 @@ def predict(self, data):
 def _contains_pdparams(path):
     file_list = os.listdir(path)
     return any(".pdparams" in file for file in file_list)
+
+
+@experimental
+@autologging_integration(FLAVOR_NAME)
+def autolog(
+    optimizer=None,
```

Comment:
Can we obtain `optimizer` from an `paddle.Model` instance?

Reply:
Currently no, we can get network info from `paddle.Model` but not `optimizer`.